modules/statics/src/coins.ts

```diff
@@ -108,6 +108,7 @@ export const coins = CoinMap.fromCoins([
     ETH_FEATURES_WITH_STAKING
   ),
   account('eth2', 'Ethereum 2.0', Networks.main.ethereum2, 18, UnderlyingAsset.ETH2, ETH2_FEATURES, KeyCurve.BLS),
+  account('ethw', 'Ethereum PoW', Networks.main.ethereumW, 18, UnderlyingAsset.ETHW, [...AccountCoin.DEFAULT_FEATURES]),
```

Comment:
nit: do we want ETH_FEATURES?

Reply:
`ETH_FEATURES` adds these two features to the account default features, `SUPPORTS_TOKENS` and `ENTERPRISE_PAYS_FEES`. Since we only want to support recovery, I think we don't need them. Is that right?